prac_05/emails.py

```diff
@@ -0,0 +1,23 @@
+def main():
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name_from_email(email)
+        confirmation = input("Is your name {}? (Y/n ".format(name))
+        if confirmation.upper() != "Y" and confirmation != "":
```

Comment:
Did you have a reason for making != "Y", and != "". instead of = "n".

Reply:
No reason, your way is actually better thank you